src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -258,5 +258,6 @@ export const walletModel = {
     walletRemoved,
     walletRemovedSuccess: removeWalletFx.done,
     walletsRemoved,
+    proxiedWalletsCreatedSuccess: proxiedWalletsCreatedFx.done,
```

Comment:
Please move `proxiedWalletsCreatedFx` into `proxies-model`
There is also a todo:
```javascript
// TODO: Move wallet creation to it's own feature
const proxiedWalletsCreatedFx = createEffect(
  (proxiedWallets: CreateParams<ProxiedAccount>[]): Promise<(CreateResult | undefined)[]> => {
    return Promise.all(proxiedWallets.map(walletCreatedFx));
  },
);
```